dtbase/backend/README.md

```diff
@@ -1,18 +1,33 @@
-# API documentation
+# DTBase Backend
 
-The DTBase backend is a FastAPI application providing an API for interacting with the database.
-This file used to list all the end points in the API.
-That functionality has now been taken over by the backend docs page.
-If you are developing/running locally, and your backend is running at `http://localhost:5000`, you can find the docs at `http://localhost:5000/docs`.
-Correspondingly for an Azure deployment it will be something like `https://<your-azure-app-name>_backend.azurewebsites.net/docs`.
-The docs are automatically generated from the code, and will list all the end points, the methods and payloads they accept, and what they return.
+The backend is the heart of DTBase. The frontend is just a pretty wrapper for the backend, and all the services are just scripts that call various backend API endpoints. All data is stored in the PostgreSQL database, and the backend is the only recommended way to access that database.
 
-## Authentication
+The backend is a web app implemented using FastAPI. It takes in REST requests and returns responses.
 
-To be able to access any of the API end points you need an authentication token.
-You can get one from the `/auth/login` endpoint using a username and a password.
-Once you've obtained a token, you need to add it to header of any other API calls you make as a bearer token.
-So if `/auth/login` returned
+## Code structure
+* `run.sh`. This is script you call to run the FastAPI app.
+* `run_localdb.sh`. Just like `run.sh` except sources a different file of secrets and ensures that a local PostgreSQL server is running.
```

Comment:
Lets try and reduce this down to a single run.sh file when I merge the local installation.